atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -88,7 +89,18 @@ private AtlasDbHttpClients() {
      */
     public static <T> T createProxyWithFailover(
             Optional<SSLSocketFactory> sslSocketFactory, Collection<String> endpointUris, Class<T> type) {
-        FailoverFeignTarget<T> failoverFeignTarget = new FailoverFeignTarget<>(endpointUris, type);
+        return createProxyWithFailover(sslSocketFactory, endpointUris, new Request.Options(),
+                FailoverFeignTarget.DEFAULT_MAX_BACKOFF_MILLIS, type);
+    }
+
+    /**
+     * As above, but you can optionally override Feign options, and the maximum time that the failover client will
+     * backoff for.
```

Comment:
nit: I'd prefer for us to not use "as above", because Javadoc has a rather interesting way of sorting its methods (it sorts them alphabetically and then by types). Also IntelliJ's autocompletion (Ctrl-J for me) has its own ordering too, so it may be confusing to see "as above" without the context.